### DISCUSSION TITLE
Unit of measurement for getPlaytime

### DISCUSSION THREAD

**Discussion by susps** [danielwerg/r6api.js#54] [Q&A] [answered]
Hey, so I found your API on NPM, and was confused on what unit the value is when requesting getPlaytime

Thanks!

**Comment by danielwerg** [comment]
I read your question several time but I still don't understand what are you asking. You can see [here](https://github.com/danielwerg/r6api.js#getPlaytime) that parameters `getPlaytime` takes and expected output.

**Comment by EndBug** [comment, marked as answer]
@danielwerg I think that they want to know what does the integer that is associated with each field represent. Correct me if I'm wrong, but it should be seconds, so: `4402996` = 4402996 seconds = ~1223 hours

**Reply by danielwerg** [reply]
If that was their question then yeah playtime is stored in seconds.
